### ISSUE TITLE
Resolution / menu's is wrong

### ISSUE THREAD

**Issue by GoogleCodeExporter** [billycris007/open-gpstracker#74]
```
* After a relaunch the resolution if of
* like if it is trying to draw it on a smaller screen or something.
* the font size there is very, very small, unreadable
* the menu buttons also "shrink"

This didn't happen before the 0.9.13 upgrade

Affected: 
*Motorola Milestone
*Nexus One

Working:
*Italian Motorola Milestone, Android 2.0.1 
```

Original issue reported on code.google.com by `rcgr...@gmail.com` on 21 Feb 2010 at 10:14

**Comment by GoogleCodeExporter** [comment]
Original comment by `rcgr...@gmail.com` on 21 Feb 2010 at 10:15
- Added labels: **Priority-Critical**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Upgrade the app to Android 1.6 and added some high density icons. Hopefuly 
avoiding
the screen emulation fixes screen shrinking.... 
```

Original comment by `rcgr...@gmail.com` on 21 Feb 2010 at 8:36
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed by 1.6 upgrade
```

Original comment by `rcgr...@gmail.com` on 22 Feb 2010 at 5:16
- Changed state: **Fixed**
